### PR TITLE
Migrate netlify-cms to decap-cms

### PR DESCRIPTION
**Pull request by anselmbradford**
netlify-cms has migrated to decap-cms https://www.netlify.com/blog/netlify-cms-to-become-decap-cms/

## Changes

- Migrate netlify-cms to decap-cms

## Testing

1. `yarn start` should run a local server at localhost:4000/design-system, where you can edit pages.

### CODE REVIEW

File: docs/assets/css/netlify.scss [hunk 1]
```diff
@@ -30,7 +30,13 @@
...
+// Don't show the toggle code button in the Decap CMS preview pane
 .frame-content #toggle-code {
   display: none;
 }
+
+// By default the preview pane doesn't have any padding.
+.frame-content {
+  padding-left: 20px;
+  padding-right: 20px;
+}
```
Comment (contolini) [inline docs/assets/css/netlify.scss:42]: Can this entire file be deleted now that `decap.scss` exists? And import it in [`main.scss`](https://github.com/cfpb/design-system/blob/4dd8418fa6d637bb74de770fb3c8a4bbc4e74f1d/docs/assets/css/main.scss#L33).

File: docs/assets/js/admin/decap-cms.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
-import CMS from 'netlify-cms';
+import CMS from 'decap-cms-app';
 import { StringTrimmedControl } from './widgets/StringTrimmed.js';
 import { StringWithInstructionsControl } from './widgets/StringWithInstructions.js';
 import genericPreviewTemplate from './widgets/genericPreviewTemplate.js';
 import navigationPreviewTemplate from './widgets/navigationPreviewTemplate.js';
 import pagePreviewTemplate from './widgets/pagePreviewTemplate.js';
 
+CMS.init();
```
Comment (contolini) [inline docs/assets/js/admin/decap-cms.js:8]: It's weird that we need to initialize the CMS when the [docs](https://decapcms.org/docs/manual-initialization/) state that it's only necessary when the `CMS_MANUAL_INIT` flag is enabled. 🤷

Comment (anselmbradford) [reply docs/assets/js/admin/decap-cms.js:8]: That is weird! But if I remove it the CMS screen is a blank screen 🤷

File: docs/assets/css/netlify.scss (hunk 1 above)
Comment (anselmbradford) [reply docs/assets/css/netlify.scss:42]: Mmm yes indeed https://github.com/cfpb/design-system/pull/1980/commits/d8423951840088e961ad9c0811b22ef033555236

**Review by contolini — APPROVED** [review]
🚀

**Pull request merged**